docs/workers.md

```diff
@@ -347,6 +368,25 @@ stream_writers:
     events: event_persister1
 ```
 
+Each of the streams have associated endpoints which should have `POST` and `PUT`
+requests routed to the workers handling that stream. Otherwise, those requests
+will be proxied to the proper worker.
+
+See below for the currently supported streams and the endpoints associated with
+them:
+
+##### The `events` stream
+
+The following endpoints should be routed direclty to the workers configured as
+stream writers for the `events` stream:
+
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/redact
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/send
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/state/
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/(join|invite|leave|ban|unban|kick)$
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/join/
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/profile/
```

Comment:
I don't think this is right. It's at least not what we do on matrix.org. These requests come in to a generic worker (in the form of the `event_creator`s), which does a bunch of work before passing them onto the things handling the events streams (the `event_persister`s).

Reply:
Oh, so the `event_persister` itself doesn't handle any endpoints! 👍